# Results

+ ^1.Plasmid construction^

  + ^1.1 the plasmids required for fusion proteins expression^

    + ^1.1.1 pYES2-PGAL1-Pir1p-SNAP25-(TEV)-Mating factor α-CYC1(pYES2-TEVSITE)^

      In the Substrate acting sub strain, we constructed a single fusion-protein expression plasmid, pYES2-PGAL1-Pir1p-SNAP25-(TEV)-Mating factor α-CYC1(pYES2-TEVSITE), assembled from the Pir1p, SNAP-25 and (TEV)-mating-factor α coding sequences. Expression of this fusion protein (Pir1p-SNAP25-(TEV)-Mating factor α, termed substrate probe) in the Substrate Acting Strain establishes the required cell-surface display system. 

      ![Map of pYES2-PGAL1-Pir1p-SNAP25-(TEV)-Mating factor alpha-CYC1.](url_of_Fig1)

      ^Figure 1:^ Map of pYES2-PGAL1-Pir1p-SNAP25-(TEV)-Mating factor alpha-CYC1.

      During construction, the Pir1p fragment was readily amplified. However, we found that the (TEV)-mating-factor alpha segment could not be efficiently generated by standard PCR and was too short for direct Gibson assembly. We therefore employed overlap PCR to produce a SNAP25-TEVsite-α-factor fusion fragment and subsequently introduced this product into the Gibson recombination reaction.

      ![The PCR result of SNAP25-TEVsite-alpha-factor fusion fragment. The band was identical to the expected length of 251bp.](url_of_Fig2)

      ^Figure 2:^ The PCR result of SNAP25-TEVsite-alpha-factor fusion fragment. The band was identical to the expected length of 251bp.

      The PCR result of SNAP25-TEVsite-alpha-factor fusion fragment. The band was identical to the expected length of 251bp.

      ![The colony PCR result of pYES2-PGAL1-Pir1p-SNAP25-(TEV)-Mating factor alpha-CYC1 plasmid. The band was identical to the expected length of 1451bp.](url_of_Fig3)

      ^Figure 3:^ The colony PCR result of pYES2-PGAL1-Pir1p-SNAP25-(TEV)-Mating factor alpha-CYC1 plasmid. The band was identical to the expected length of 1451bp.

    + ^1.1.2 pUC19-sTF^

      To construct the signal-transferring strain, we aimed to replace the DNA-binding domain (DBD) of the Ste12 transcription factor with that of Gal4, thereby coupling the Ste2 receptor to the P<sub>GAL</sub> promoter. So we built the pUC19-sTF plasmid. The donor fragment amplified from this plasmid contains the Gal4-DBD coding sequence flanked by 300–600 bp homology arms on both sides. Upon transformation into Saccharomyces cerevisiae BY4741, the fragment directs homologous recombination that substitutes the STE12 DBD with the Gal4 DBD.

      ![structure of pUC19-sTF.](url_of_Fig4)

      ^Figure 4:^ structure of pUC19-sTF.

      The construction of pUC19-sTF has multiple steps, including construction of the pUC19-*STE12*, and replacement of STE12DBD.

      The pUC19-*STE12* carried the *STE12* fragment amplificated from Saccharomyces cerevisiae BY4741.

      ![Plasmid map of pUC19-STE12.](url_of_Fig5)

      ^Figure 5:^ Plasmid map of pUC19-*STE12*.

      ![a. Colony PCR results after pUC-STE12 construction and transformation showed the successful the plasmid construction. b. the fragment map from shows the designment of primer used in the colony PCR and the length of the amplification.](url_of_Fig6)

      ^Figure 6:^ a. Colony PCR results after pUC-*STE12* construction and transformation showed the successful the plasmid construction. b. the fragment map from shows the designment of primer used in the colony PCR and the length of the amplification.

      ![a. Colony PCR results after pUC-sTF construction and transformation showed the successful the plasmid construction. b. the fragment map from shows the designment of primer used in the colony PCR and the length of the amplification. ](url_of_Fig7)

      ^Figure 7:^ a. Colony PCR results after pUC-sTF construction and transformation showed the successful the plasmid construction. b. the fragment map from shows the designment of primer used in the colony PCR and the length of the amplification.

  + ^1.2 Construction of plasmids required for yeast gene knockout^

    In order to modify Botulinum Toxin Yeast-based biosensor, we constructed four plasmids for knocking out yeast genes, including pUC19-*GAL4-URA3*, pUC19-*GAL80-HIS3*, pUC19-*STE2-URA3*, pUC19-*FAR1-URA3*. Each plasmid contains donor fragment composed of a yeast auxotrophic marker gene, and there are homologous sequences of 400-500 bp on each side of the knocked out gene. Afterwards, we obtained the corresponding fragments through PCR and transformed them into yeast cells for gene knockout through homologous recombination.

    ![a. Map of plasmid pUC19-GAL4-URA3.  b. Map of plasmid pUC19-GAL80-HIS3 .c. Map of plasmid pUC19-STE2-URA3. d. Map of plasmid pUC19-FAR1-URA3.](url_of_Fig8)

    ^Figure 8:^ a. Map of plasmid pUC19-*GAL4-URA3*. b. Map of plasmid pUC19-*GAL80-HIS3*. c. Map of plasmid pUC19-*STE2-URA3*. d. Map of plasmid pUC19-*FAR1-URA3*.

    ![The PCR result of GAL4-URA3 fragment. The band was identical to the expected length of 2297 bp.](url_of_Fig9)

    ^Figure 9:^ The PCR result of *GAL4-URA3* fragment. The band was identical to the expected length of 2297 bp.

    ![The PCR result of GAL80-HIS3 fragment. The band was identical to the expected length of 2039 bp.](url_of_Fig10)

    ^Figure 10:^ The PCR result of *GAL80-HIS3* fragment. The band was identical to the expected length of 2039 bp.

    ![The PCR result of STE2-URA3 fragment. The band was identical to the expected length of 2425 bp.](url_of_Fig11)

    ^Figure 11:^ The PCR result of *STE2-URA3* fragment. The band was identical to the expected length of 2425 bp.

    ![The PCR result of URA3 fragment. The band was identical to the expected length of 1449 bp.](url_of_Fig12)

    ^Figure 12:^ The PCR result of *URA3* fragment. The band was identical to the expected length of 1449 bp.

  + ^1.3 the plasmids required for visible output^

    To preliminarily verify the feasibility of the signal system, we constructed the plasmid pYES2-PGAL1-*MjDOD*-CYC1. This plasmid contains the GAL1 promoter, which allows yeast to express *MjDOD* enzyme under lactose induction. This enzyme can convert L-DOPA into a yellow pigment.

    ![Map of plasmid pYES2-PGAL1-MjDOD-CYC1](url_of_Fig13)

    ^Figure 13:^ Map of plasmid pYES2-PGAL1-*MjDOD*-CYC1.

    ![The PCR result of MjDOD fragment. The band was identical to the expected length of 862 bp](url_of_Fig14)

    ^Figure 14:^ The PCR result of *MjDOD* fragment. The band was identical to the expected length of 862 bp.

+ ^2.Engineering Renovation^

  Our dual-yeast biosensor for TEV or BoNT/A protease activity comprises two engineered strains: the Substrate Acting Strain expressing the substrate probe and carries a *STE2* knockout and the Signal Transffering Strain expressing MjDOD and sTF while lacking GAL4, GAL80, FAR1 and STE12.

  + ^2.1 Substrate Acting Strain Construction^

    We transformed the donor fragment from plasmid pUC19-*STE2*-URA3 into *Saccharomyces cerevisiae* BY4741 and colony PCR verified the correct homologous recombination, confirming successful disruption of the target gene by marker insertion.

    ![Saccharomyces cerevisiae BY4741 colony PCR results of STE2-URA3. The band was identical to the expected length of 2425 bp](url_of_Fig15)

    ^Figure 15:^ *Saccharomyces cerevisiae* BY4741 colony PCR results of *STE2*-URA3. The band was identical to the expected length of 2425 bp.

    The transformation of the plasmid pYES2-PGAL1-Pir1p-SNAP25-(TEV)-Mating factor α-CYC1(pYES2-TEVSITE) and the fusion protein substrate probe expression verification are still tasks in progress as far as the Wiki freeze and will be finished before the Jamboree.

  + ^2.2 Signal Transffering Strain Construction^

    To construct the *GAL4GAL80*-knockout yeast, we transformed the donor fragment from pUC19-*GAL4-URA3*, pUC19-*GAL80-HIS3* and selected the yeast with correct homologous recombination by colony PCR.

    ![a. Saccharomyces cerevisiae BY4741 colony PCR results of GAL80-HIS3. b. Saccharomyces cerevisiae BY4741 colony PCR results of GAL4-URA3.](url_of_Fig16)

    ^Figure 16:^ a. *Saccharomyces cerevisiae* BY4741 colony PCR results of *GAL80-HIS3*. b. *Saccharomyces cerevisiae* BY4741 colony PCR results of *GAL4-URA3*.

    To couple the Mating factor alpha-dependent signal transduction and the expression of MjDOD, we need to swap the STE12-DBD with GAL4-DBD to construct the fusion protein sTF which responses the signal transduction by upstream of the circuit and triggers the expression of MjDOD.

    We transformed the donor fragment from the plasmid pUC19-sTF and the correct homologous recombination was verified by diagnostic PCR analysis later on.

    ![Donor DNA used for DBD replacement of the STE12 gene](url_of_Fig17)

    ^Figure 17:^ Donor DNA used for DBD replacement of the STE12 gene.

    ![A. the results of diagnostic PCR analysis. The samples from Colonies 1–16 of plate a and colonies 1–9 plus 11–16 of plate b carried the right homologous recombination. B. the fragment map from shows the designment of primer used in the colony PCR and the length of the amplification.](url_of_Fig18)

    ^Figure 18:^ A. the results of diagnostic PCR analysis. The samples from Colonies 1–16 of plate a and colonies 1–9 plus 11–16 of plate b carried the right homologous recombination. B. the fragment map from shows the designment of primer used in the colony PCR and the length of the amplification.

+ ^3.Visible Output Preliminarily Verification^

  To preliminarily verify the feasibility of the signal system, we transformed the plasmid pYES2-PGAL1-*MjDOD*-CYC1 into yeast. Then, we inoculated yeast into SG medium and added 0.1 M L-DOPA and 1 M ascorbic acid to the medium for 48 hours of cultivation. Betaxanthin can be clearly seen in the supernatant of the culture medium, which confirming the expression of MjDOD.

  ![The synthesis of betaxanthin. From left to right are blank culture medium, yeast without transformed plasmids, and two yeast strains transformed with pYES2-PGAL1-MjDOD-CYC1 plasmids](url_of_Fig19)

  ^Figure 19:^ The synthesis of betaxanthin. From left to right are blank culture mediums, yeast without transformed plasmids, and two yeast strains transformed with pYES2-PGAL1-*MjDOD*-CYC1 plasmids.

  ![Color of cells and culture medium](url_of_Fig20)

  ^Figure 20:^ Color of cells and culture medium.



